src/metabase/query_processor/middleware/format_rows.clj

```diff
@@ -60,14 +60,20 @@
     (t/format :iso-offset-date-time
               (t/offset-date-time (t/with-zone-same-instant t timezone-id)))))
 
+(defn map-row-builder
```

Comment:
Why did you decide to factor out this function? If you wanted to make sure you get a lazy sequence out, `for` should already do that.

Reply:
Good catch, this snuck in from the original PR and is no longer necessary.